### ISSUE TITLE
Broken on react-native

### ISSUE THREAD

**Issue by machour** [styled-components/styled-components#1456]
## Version
styled-components: 3.1.0

## Reproduction
Try to use 3.1.0 in a React Native application, packager will error the following:

```
error: bundling failed: Error: Unable to resolve module `../dist/styled-components.cjs.native` from `/Users/didou/js/react-native/git-point/node_modules/styled-components/native/index.js`: The module `../dist/styled-components.cjs.native` could not be found from `/Users/didou/js/react-native/git-point/node_modules/styled-components/native/index.js`. Indeed, none of these files exist:

  * `/Users/didou/js/react-native/git-point/node_modules/styled-components/dist/styled-components.cjs.native(.native||.android.js|.native.js|.js|.android.json|.native.json|.json)`
  * `/Users/didou/js/react-native/git-point/node_modules/styled-components/dist/styled-components.cjs.native/index(.native||.android.js|.native.js|.js|.android.json|.native.json|.json)`
    at ModuleResolver.resolveDependency (/Users/didou/js/react-native/git-point/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:311:1122)
```

It seems that the file is actually named `styled-components.native.js` in dist (conforming to rollup.config) and that this bug was introduced in this commit: https://github.com/styled-components/styled-components/commit/8cf3cb191a51223a173c95f94d653448af5b7196

## Steps to reproduce

Install 3.1.0 in a React Native application and start it

## Expected Behavior

App should start

## Actual Behavior

Packager error

**Comment by machour** [comment]
Was about to submit a PR, but just stumbled upon #1455 :)

**Comment by mxstbr** [comment]
Shipping that PR as `v3.1.1`!

**Comment by mxstbr** [comment]
Fixed in `v3.1.1`, sorry about the short-lived troubles! Thanks to @vvasilev- for the quick PR :heart:
